### DISCUSSION TITLE
Criar links de url com base em um useState

### DISCUSSION THREAD

**Discussion by felipevc13** [frontendbr/forum#1962] [Perguntas e Respostas] [answered]
Fala pessoal,
Boa noite.

Como eu faço pra criar links de url com base em um useState? 
Preciso criar os links com base em um datepicker de calendário.

<img width="695" alt="Captura de Tela 2020-12-23 às 18 54 16" src="https://user-images.githubusercontent.com/25945473/103039838-9ab22b00-4550-11eb-8c5f-495cb2ae42a5.png">

**Comment by victorferreira** [comment, marked as answer]
Não sei se entendi bem a sua dúvida, mas parece que o que você precisa é de um roteador e usar os paramentros passados na URL. Acho que o [react-router](https://github.com/ReactTraining/react-router) tem o que você precisa. Dá uma olhada nos [docs](https://reactrouter.com/web/example/url-params).

**Reply by felipevc13** [reply]
Muito obrigado pela resposta Victor, como estou usando o gatsby acabei usando esse navigate que praticamente eh o router dele. Vlww pela dica.
<img width="554" alt="Captura de Tela 2020-12-28 às 19 30 55" src="https://user-images.githubusercontent.com/25945473/103246737-48ec1500-4943-11eb-95a4-81c2d12f8cb5.png">
